api/src/main/java/keywhiz/api/model/SecretSeries.java

```diff
@@ -46,20 +46,23 @@ public static SecretSeries of(
       @Nullable String type,
       @Nullable Map<String, String> generationOptions,
       @Nullable Long currentVersion) {
+
     ImmutableMap<String, String> options = (generationOptions == null) ?
         ImmutableMap.of() : ImmutableMap.copyOf(generationOptions);
-    return new AutoValue_SecretSeries(
-        id,
-        name,
-        owner,
-        nullToEmpty(description),
-        createdAt,
-        nullToEmpty(createdBy),
-        updatedAt,
-        nullToEmpty(updatedBy),
-        Optional.ofNullable(type),
-        options,
-        Optional.ofNullable(currentVersion));
+
+    return builder()
+        .id(id)
+        .name(name)
+        .owner(owner)
```

Comment:
We want to keep this as "owner" because this is part of the external facing API, right?

Reply:
Yeah, and I think that makes sense because the owner name and ID are only distinct internally because of the database model, externally there is only the owner name (the internal data ID is never exposed) so leaving it as just owner makes more sense. Although, we could still call it ownerName, but we'd have to change some APIs (not the end of the world, just more work).

Reply:
Yeah, that makes sense. If the API only uses the owner name, then we can use "owner" as a shorthand without causing any confusion.